frontend/src/main/components/PersonalSchedules/PersonalScheduleEvent.js

```diff
@@ -0,0 +1,123 @@
+import React, { useEffect, useState } from "react";
+import {
+  ButtonGroup,
+  Card,
+  OverlayTrigger,
+  Popover,
+  Button,
+} from "react-bootstrap";
+
+export default function SchedulerEvents({ event, eventColor, borderColor }) {
+  const [style, setStyle] = useState({});
+
+  const testId = "SchedulerEvent";
+
+  const convertTimeToMinutes = (time) => {
+    const [timePart, modifier] = [time.slice(0, -2), time.slice(-2)];
+    let [hours, minutes] = timePart.split(":").map(Number);
+
+    if (modifier === "PM" && hours !== 12) {
+      hours += 12;
+    } else if (modifier === "AM" && hours === 12) {
+      hours = 0;
+    }
+
+    return hours * 60 + minutes;
+  };
+
+  // Stryker disable all : hard to test for specfic styles
+  useEffect(() => {
+    const startMinutes = convertTimeToMinutes(event.startTime);
+    const endMinutes = convertTimeToMinutes(event.endTime);
+    const height = endMinutes - startMinutes;
+    const topPosition = startMinutes + 94;
```

Comment:
The `// Stryker disable all : hard to test for specfic styles` still has code inside of it that is NOT style code.

This is not acceptable.  If you are claiming an exception, it has to be narrowly tailored to *only* exclude lines of code that you have a legitimate reason to not test.   Correctness *depends* on this code; it's not just a matter of what color or font.   So please move the stryker exception below this code, and write tests for this code.